portfolio-project-matching-app/Firebase/auth.js

```diff
@@ -0,0 +1,39 @@
+import { getAuth, signInWithPopup, GoogleAuthProvider, onAuthStateChanged, signOut, auth } from "firebase/auth";
+import { useAuthUpdate } from '../context/AuthContext';
+import { firebaseApp } from "./clientApp.ts";
```

Comment:
Nice work here. I was dumb enough to plug all the Firebase Firestore Database stuff directly into `clientApp.ts`. Either too dumb, or just lacked the perspective when I was getting things setup. I should probably move the DB stuff into a dedicated file like you did.